lib/runtime/wasmer/exports_test.go

```diff
@@ -408,6 +408,32 @@ func TestInstance_GrandpaAuthorities_PolkadotRuntime(t *testing.T) {
 	require.Equal(t, expected, auths)
 }
 
+func TestInstance_BabeGenerateKeyOwnershipProof_NodeRuntime(t *testing.T) {
+	tt := trie.NewEmptyTrie()
+	rt := NewTestInstanceWithTrie(t, runtime.NODE_RUNTIME, tt)
+	authorityID := types.AuthorityID{}
+	slot := uint64(1)
+	_, err := rt.BabeGenerateKeyOwnershipProof(slot, authorityID)
+	require.NoError(t, err)
+}
+
+func TestInstance_BabeSubmitReportEquivocationUnsignedExtrinsic_NodeRuntime_(t *testing.T) {
+	tt := trie.NewEmptyTrie()
+	rt := NewTestInstanceWithTrie(t, runtime.NODE_RUNTIME, tt)
+	authorityID := types.AuthorityID{}
+	slot := uint64(1)
+	keyOwnershipProof, err := rt.BabeGenerateKeyOwnershipProof(slot, authorityID)
```

Comment:
The returned value is always an empty byte slice, can't we somehow use another runtime to test with actual values?

Reply:
> The returned value is always an empty byte slice, can't we somehow use another runtime to test with actual values?

So, authorityID has to be part of authorities in order to see a non empty proof. Our current runtime instance is created without authorities. I have tried changing some basic things like `role`, but that hasn't work. Will look more into it.

Once I can create a runtime instance with babe authorities, I can use ` rt.BabeConfiguration()` to get authorities and can use one of them to create a proof. Currently ` rt.BabeConfiguration()` gives you no authorities.

Reply:
I have added another copy of these tests though, with Polkadot runtime.

Reply:
Hmm, Ok.
This is definitely a problem. I ran a network and modified the code to see values of key Ownership and I am always seeing empty bytes. 
But I haven't figured out yet why this is happening.